recipes/_configure_server.rb

```diff
@@ -95,12 +96,46 @@
   end
 end
 
+path = "#{node['splunk']['home']}/etc/system/local/server.conf"
+old_stanzas = CernerSplunk::Conf::Reader.new(path).read if File.exist?(path)
```

Comment:
While true, I didn't want it inside the if block, as that only applies to SHClusters, but reading a value from existing configuration and pulling it forward is more of a general issue if that makes sense.